OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -332,6 +334,83 @@ public async Task UpdateWorkshop_WhenModelIsInvalid_ShouldReturnBadRequestObject
         //}
         #endregion
 
+        #region UpdateWorkshopStatus
+        [Test]
+        public async Task UpdateStatus_WhenModelIsValid_ShouldReturnOkObjectResult()
+        {
+            // Arrange
+            providerServiceMoq.Setup(x => x.GetByUserId(It.IsAny<string>(), It.IsAny<bool>())).ReturnsAsync(provider);
+
+            var updateRequest = new WorkshopStatusDto()
+            {
+                WorkshopId = workshop.Id,
+                Status = WorkshopStatus.Open,
+            };
+
```

Comment:
var updateRequest = new WorkshopStatusDto()
            {
                WorkshopId = workshop.Id,
                Status = WorkshopStatus.Open,
            };
            
            You use the same code some times. Will be better move it to the another method and add needed status as the method's parameter.